src/frontend/grid/index.ts

```diff
@@ -102,8 +102,11 @@ export async function run(memberlist:CluckMember[]) {
 	addEventListener("resize", redrawRows);
 })();
 
+export let timeLoggedIn;
+
 async function refreshLoggedIn() {
 	let membersIn:LoggedIn;
+	timeLoggedIn = Date.now();
```

Comment:
It looks like this is never used in any file

Reply:
oh ya, that was from when I started to re-write the circle-sizing system but never got around to it